scheme/meep.scm.in

```diff
@@ -1353,7 +1353,7 @@
   (define all-freqs '())
   (map (lambda (k)
 	 (set! k-index (+ k-index 1))
-	 (if (= k-index 1) (begin (init-fields) (output-epsilon)))
+	 (if (= k-index 1) (begin (change-k-point! k) (init-fields) (output-epsilon)))
```

Comment:
`(init-fields)` is no longer required since the fields are initialized by `change-k-point`